gstreamer/src/toc.rs

```diff
@@ -237,6 +247,20 @@ impl ToOwned for TocEntryRef {
     }
 }
 
+impl fmt::Debug for TocEntryRef {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("TocEntry")
+            .field("entry_type", &self.get_entry_type())
+            .field("uid", &self.get_uid())
+            .field("start_stop", &self.get_start_stop_times())
+            .field("tags", &self.get_tags())
+            .field("is_alternative", &self.is_alternative())
+            .field("is_sequence", &self.is_sequence())
+            .field("loop", &self.get_loop())
```

Comment:
Sub entries here?

Reply:
I don't know. In that case, this should be recursive. I though the user would choose how and when to traverse the tree.

Reply:
Debug impls usually recurse into all their children

Reply:
Ok then.